intro/scipy/index.rst

```diff
@@ -361,160 +361,90 @@ Finding the minimum of a scalar function
-
-|
-
-**Methods**:
-As the function is a smooth function, gradient-descent based methods are
-good options. The `lBFGS algorithm
-<https://en.wikipedia.org/wiki/Limited-memory_BFGS>`__ is a good choice
-in general::
-
-
-    >>> sp.optimize.minimize(f, x0=0, method="L-BFGS-B") # doctest: +ELLIPSIS
-      message: CONVERGENCE: NORM_OF_PROJECTED_GRADIENT_<=_PGTOL
-      success: True
-       status: 0
-          fun: -7.9458233756...
-            x: [-1.306e+00]
-          nit: 5
-          jac: [-1.599e-06]
-         nfev: 12
-         njev: 6
-     hess_inv: <1x1 LbfgsInvHessProduct with dtype=float64>
-
-Note how it cost only 12 functions evaluation above to find a good value
-for the minimum.
-
-|
-
-**Global minimum**:
-A possible issue with this approach is that, if the function has local minima,
-the algorithm may find these local minima instead of the
-global minimum depending on the initial point x0::
-
-    >>> res = sp.optimize.minimize(f, x0=3, method="L-BFGS-B")
-    >>> res.x
-    array([3.83746...])
-
-.. Comment to make doctest pass
-   >>> np.random.seed(42)
-
-If we don't know the neighborhood of the global minimum to choose the
-initial point, we need to resort to costlier global optimization.  To
-find the global minimum, we use :func:`scipy.optimize.basinhopping`
-(added in version 0.12.0 of Scipy). It combines a local optimizer with
-sampling of starting points::
-
-   >>> sp.optimize.basinhopping(f, 0)  # doctest: +SKIP
-                     nfev: 1725
-    minimization_failures: 0
-                      fun: -7.9458233756152845
-                        x: array([-1.30644001])
-                  message: ['requested number of basinhopping iterations completed successfully']
-                     njev: 575
-                      nit: 100
-
-.. seealso:
-
-    Another available (but much less efficient) global optimizer is
-    :func:`scipy.optimize.brute` (brute force optimization on a grid).
-
-    More algorithms for different classes of global optimization problems
-    exist, but this is out of the scope of :mod:`scipy`.  Some useful
-    packages for global optimization are OpenOpt, IPOPT_, PyGMO_ and
-    PyEvolve_.
-
-.. note::
-
-   :mod:`scipy` used to contain the routine `anneal`, it has been removed in
-   SciPy 0.16.0.
-
-.. _IPOPT: https://github.com/xuy/pyipopt
-.. _PyGMO: https://esa.github.io/pygmo/
-.. _PyEvolve: https://pyevolve.sourceforge.net/
-
-|
-
-**Constraints**:
-We can constrain the variable to the interval
-``(0, 10)`` using the "bounds" argument:
-
-.. sidebar:: A list of bounds
-
-   As :func:`~scipy.optimize.minimize` works in general with x
-   multidimensionsal, the "bounds" argument is a list of bound on each
-   dimension.
-
-::
-
-    >>> res = sp.optimize.minimize(f, x0=1,
-    ...                         bounds=((0, 10), ))
-    >>> res.x    # doctest: +ELLIPSIS
-    array([0.])
-
-.. tip::
-
-   What has happened? Why are we finding 0, which is not a mimimum of our
-   function.
```

Comment:
The point here was that the minimum was at the boundary due to the choice of bounds. Is such an example needed in the introductory tutorial?